packages/@lightningjs/ui-components/src/components/Key/Key.stories.js

```diff
@@ -32,42 +32,42 @@ export default {
   }
 };
 
-export const Basic = () =>
-  class Basic extends lng.Component {
-    static _template() {
-      return {
-        Key: {
-          type: KeyComponent
-        }
-      };
-    }
-  };
+// export const Basic = () =>
+//   class Basic extends lng.Component {
+//     static _template() {
+//       return {
+//         Key: {
+//           type: KeyComponent
+//         }
+//       };
+//     }
+//   };
 
-Basic.parameters = {};
-Basic.args = {
-  title: 'A',
-  size: 'sm',
-  mode: 'focused'
-};
+// Basic.parameters = {};
+// Basic.args = {
+//   title: 'A',
+//   size: 'sm',
+//   mode: 'focused'
+// };
 
-Basic.argTypes = {
-  ...createModeControl({ summaryValue: 'focused' }),
-  size: {
-    control: 'select',
-    options: ['sm', 'md', 'lg', 'xl'],
-    description: 'Width of the Key',
-    table: {
-      defaultValue: { summary: 'sm' }
-    }
-  },
-  title: {
-    control: 'text',
-    description: 'Key character',
-    table: {
-      defaultValue: { summary: 'undefined' }
-    }
-  }
-};
+// Basic.argTypes = {
+//   ...createModeControl({ summaryValue: 'focused' }),
+//   size: {
+//     control: 'select',
+//     options: ['sm', 'md', 'lg', 'xl'],
+//     description: 'Width of the Key',
+//     table: {
+//       defaultValue: { summary: 'sm' }
+//     }
+//   },
+//   title: {
+//     control: 'text',
+//     description: 'Key character',
+//     table: {
+//       defaultValue: { summary: 'undefined' }
+//     }
+//   }
+// };
```

Comment:
Is this still in-progress, can it be removed, or is it to circle back to in a separate PR?

Reply:
Fix, thanks!